www_src/pages/project/project.jsx

```diff
@@ -139,22 +139,55 @@ var Project = React.createClass({
       currentX = x + (event.touches[0].clientX - startX);
       currentY = y + (event.touches[0].clientY - startY);
       translateStr = 'translate(' + currentX + 'px, ' + currentY + 'px)';
-      boundingEl.style.transform = translateStr + ' ' + scaleStr;
+
+      endX = event.touches[0].clientX;
+      endY = event.touches[0].clientY;
+
+      if (!this.state.isPageZoomed) {
+        boundingEl.style.transform = translateStr + ' ' + scaleStr;
+      }
     });
 
     el.addEventListener('touchend', (event) => {
-      console.log('end', event.touches.length, event);
       if (event.touches.length === 0) {
         boundingEl.style.transition = '';
         if (!didMove) return;
 
-        var state = {camera: {
-          x: currentX,
-          y: currentY
-        }};
-        if (typeof currentZoom !== 'undefined') state.zoom = currentZoom;
-        this.setState(state);
-        startX, startY, startDistance, currentX, currentY, currentZoom = undefined;
+        if (!this.state.isPageZoomed) {
```

Comment:
Can you maybe flip this around so it's a bit more readable and add a comment for the not selected state?